win-virtio: 0.1.196-1 -> 0.1.229-1

ajs124 opened the pull request:
###### Description of changes
Also just copy everything instead of just some things very elaborately.
There is no clear motivation given for why this was done in 91b85236d0651983b2d2162623309f497cc0f225.

Closes #187710 cc @samuel-f 

###### Things done
- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

ajs124 commented:
cc @samuel-f

Lassulus commented:
Result of `nixpkgs-review pr 211107` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>win-spice</li>
  </ul>
</details>
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>win-virtio</li>
  </ul>
</details>

bjornfor reviewed (commented):
Please separate out the refactoring in its own commit (either before or after the version bump commit).

ajs124 committed:
win-virtio: 0.1.196-1 -> 0.1.229-1

ajs124 committed:
win-virtio: refactor

- don't use p7zip anymore
- just copy everything instead of just some things very elaborately
  there is no clear motivation given for why this was done in 91b85236d0651983b2d2162623309f497cc0f225
